src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj

```diff
@@ -19,10 +19,20 @@
     <Copyright>© Open Systems Pharmacology Community 2020</Copyright>
     <Product>OSPSuite FuncParser</Product>
     <Configurations>Debug;Release;Purify_Debug</Configurations>
+    <NuspecProperties>version=$(PackageVersion)</NuspecProperties>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
+  <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Windows'))">
```

Comment:
just curious: what's the advantage of `[MSBuild]::IsOSPlatform` vs. `'$(OS)' ==` ?

Reply:
The `$(OS)` is limited and does not distinguish Unix from Mac

https://github.com/dotnet/msbuild/issues/2468#issuecomment-324694593

Reply:
ok, thx